<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modify_supplier</class>
 <widget class="QDialog" name="modify_supplier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sửa/Xóa nhà cung cấp</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sửa/Xóa nhà cung cấp</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>271</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Mã nhà cung cấp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Tên nhà cung cấp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Địa chỉ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_sdtNCC">
        <property name="text">
         <string>Số điện thoại</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_emailNCC">
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="le_maNCC">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_tenNCC"/>
      </item>
      <item>
       <widget class="QLineEdit" name="le_dia_chi"/>
      </item>
      <item>
       <widget class="QLineEdit" name="le_sdtNCC"/>
      </item>
      <item>
       <widget class="QLineEdit" name="le_emailNCC"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tv_nha_cung_cap">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>271</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pb_sua_nha_cung_cap">
      <property name="text">
       <string>Sửa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_xoa_nha_cung_cap">
      <property name="text">
       <string>Xóa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_thoat">
      <property name="text">
       <string>Hủy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
